spec/testdumper_acceptance_spec.rb

```diff
@@ -10,14 +14,34 @@
 {"test":"1","className":"iPhoneTestClassGama","event":"end-test","totalDuration":"0"}
 {"test":"1","className":"iPhoneTestClassOmega","event":"end-test","totalDuration":"0"}
 {"message":"Completed Test Dumper","event":"end-action","testType":"LOGICTEST"}
+{"message":"Starting Test Dumper","event":"begin-test-suite","testType":"LOGICTEST"}
+{"event":"begin-ocunit","bundleName":"SwiftTestTarget.xctest","targetName":"SwiftTestTarget"}
+{"test":"1","className":"ObjCTestClass","event":"end-test","totalDuration":"0"}
+{"test":"1","className":"SwiftTestTarget","event":"end-test","totalDuration":"0"}
+{"message":"Completed Test Dumper","event":"end-action","testType":"LOGICTEST"}
+{"message":"Starting Test Dumper","event":"begin-test-suite","testType":"APPTEST"}
+{"event":"begin-ocunit","bundleName":"iPadTestTarget.xctest","targetName":"iPadTestTarget"}
```

Comment:
This is interesting. Why is this newly showing up now?

Reply:
I'm not really 100% certain without spending more time investigating. 

I would probably try to isolate this by progressively bumping the Xcode version from 9 to 10, then 10 to 11 to see which jump the issue started appearing in and work backwards from there, maybe checking out Xcode change logs for each version hop.

Reply:
I think this was caused by a missed update on sim_runtime. Seem to behave correctly now.